radix-engine/README.md

```diff
@@ -0,0 +1,21 @@
+|----------------|------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| Blueprints     | Application Logic (e.g. Blueprints written in Scrypto)                             |                                                                                                                                                                                                                                                                              |                                                          | [Native Blueprints](src/blueprints)<br>[Scrypto Blueprints](../radix-engine-tests/tests/blueprints)        | 
+| VM             | Application Execution                                                              | WASM + [Scrypto API](../scrypto/src/engine/scrypto_env.rs)                                                                                                                                                                                                                   |                                                          | [VM](src/vm)                                                                                |
+| System         | Type Checking<br>Package/Blueprint/Object semantics<br>Application Standardization | [Substate API](../radix-engine-interface/src/api/substate_api.rs)<br>[Object API](../radix-engine-interface/src/api/object_api.rs)<br>[Blueprint API](../radix-engine-interface/src/api/blueprint_api.rs)<br>[Costing API](../radix-engine-interface/src/api/costing_api.rs) | [System Callback API](src/system/system_callback_api.rs) | [System](src/system)                                                                                       |
+| Kernel         | Call Frame Logic<br>Ownership/Reference handling                                   | [Kernel API](src/kernel/kernel_api.rs)                                                                                                                                                                                                                                       | [Kernel Callback API](src/kernel/kernel_callback_api.rs) | [Kernel](src/kernel)                                                                                       |
```

Comment:
Could you explain where what was "Kernel Modules" live - are they now in the System layer? How do they interface with System?

Reply:
Yup, the modules live in the system layer so they have the kernel layer API available to them